policyengine_uk/parameters/gov/ons/population.yaml

```
@@ -4,13 +4,33 @@ values:
   2020-01-01: 67_100_000
   2021-01-01: 67_400_000
   2022-01-01: 67_600_000
-  2023-01-01: 67_800_000
-  2024-01-01: 68_100_000
-  2025-01-01: 68_300_000
-  2026-01-01: 68_500_000
-  2027-01-01: 68_700_000
+  2023-01-01: 68_500_000
```

Clarifying actuals/projections
```suggestion
  # Projections:
  2023-01-01: 68_500_000
```